### PR TITLE
Admin combohud verb

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
:cl: Naksu
admin: Admins can now toggle antag, med, sci, engineering huds and maximum ghost brightness with just one button. 
/:cl:

### CODE REVIEW

**Comment by Jalleo** [comment]
Thanks I was going to do this next week in a different manner. (While they were aghosted only)

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1028,6 +1028,35 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 	log_admin("[key_name(usr)] toggled their admin antag HUD [adding_hud ? "ON" : "OFF"].")
 	SSblackbox.record_feedback("nested tally", "admin_toggle", 1, list("Toggle Antag HUD", "[adding_hud ? "Enabled" : "Disabled"]")) //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/toggle_combo_hud()
+	set category = "Admin"
+	set name = "Toggle Combo HUD"
+	set desc = "Toggles the Admin Combo HUD (antag, sci, med, eng)"
+
+	if(!holder)
```
Comment (Jalleo) [inline code/modules/admin/verbs/randomverbs.dm]: See my PR at the moment this is not a suitable check

**Comment by Bawhoppen** [comment]
Why the max brightness though? I always felt that lowest brightness makes it clear what is dark to players ingame, while still bring clearly visible as an observer

**Comment by vuonojenmustaturska** [comment]
It was specifically requested. You can still toggle brightness once to get back to the lowest setting

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**